nilearn/experimental/surface/_maskers.py

```diff
@@ -266,20 +260,12 @@ def transform(self, img: SurfaceImage) -> np.ndarray:
     def fit_transform(self, img: SurfaceImage, y: Any = None) -> np.ndarray:
         """Prepare and perform signal extraction from regions.
 
-        Parameters
-        ----------
-        img : SurfaceImage object
-            Mesh and data for both hemispheres.
-
-        y : None
-            This parameter is unused. It is solely included for scikit-learn
-            compatibility.
+        :param img: Mesh and data for both hemispheres.
+        :param y: This parameter is unused. It is solely included for
+                  scikit-learn compatibility.
 
-        Returns
-        -------
-        numpy.ndarray
-            Signal for each element.
-            shape: (img data shape, total number of vertices)
+        :return: Signal for each element with shape
+                 (img data shape, total number of vertices).
         """
```

Comment:
SurfaceLabelsMasker.fit_transform with docstring compatible with sphinx autdoc typehints to get the types and defaults automatically added in the docstring instead of the signature (I think can also be done for the return if we want to keep it out of the signature):
![image](https://github.com/nilearn/nilearn/assets/63292494/3a428a3e-0e66-483d-8264-add20aed1701)